src/Apps/Notifications/NotificationApp.tsx

```diff
@@ -0,0 +1,33 @@
+import { FullBleed } from "@artsy/palette"
+import { MetaTags } from "Components/MetaTags"
+import { createFragmentContainer, graphql } from "react-relay"
+import { NotificationApp_me$data } from "__generated__/NotificationApp_me.graphql"
+import { NotificationFragmentContainer } from "Components/Notifications/Notification"
+
+interface NotificationAppProps {
+  me: NotificationApp_me$data
+}
+
+const NotificationApp: React.FC<NotificationAppProps> = ({ me }) => {
+  return (
+    <>
+      <MetaTags title="Notification | Artsy" pathname="/notification" />
+
+      <FullBleed>
```

Comment:
@olerichter00 - rather than wrap the component with FullBleed, you can define it via the `layout: "FullBleed"` prop directly on the route: https://github.com/artsy/force/blob/main/src/Apps/Components/Layouts/index.tsx#L23

Reply:
Thanks! 🙏